CLAUDE.md

```
@@ -2,143 +2,162 @@
+- **worker** — Celery worker for file operations (video download, PDF export, audio transcription)
 
 ## Environment Configuration
 
-### Required Variables
-- `BASE_URL`: Server base URL
-- `TELEGRAM_BOT_TOKEN`: Telegram bot token
-- `TELEGRAM_CHAT_ID`: Default chat ID for bot
+See `template.env` for a complete reference. Key variables:
+
+### Required
+| Variable | Description |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)